### DISCUSSION TITLE
Enforcing code reliability by requiring workflows with GitHub repository rules

### DISCUSSION THREAD

**Discussion by v1gnesh** [zopencommunity/meta#622] [Ideas]
https://github.blog/2023-10-11-enforcing-code-reliability-by-requiring-workflows-with-github-repository-rules/

Believe we can use this to enforce the run of our build_and_test and bump workflows.
When you have time, could you please check how this looks in this org's case @IgorTodorovskiIBM @HarithaIBM @MikeFultonDev

**Comment by IgorTodorovskiIBM** [comment]
I don't see that option, maybe it's only available in Github Enterprise as the article mentions?

We do have this option under branch protection:
<img width="938" alt="Screenshot 2023-12-12 at 10 52 53 AM" src="https://github.com/ZOSOpenTools/meta/assets/39890068/73a983b4-96f4-4c02-ad06-11ace3d43ecd">

I haven't enabled that yet until we iron out all of the issues with github actions, but once we do, I think it makes sense to enable.

**Reply by v1gnesh** [reply]
Heh, I overlooked GH E :laughing:
